### ISSUE TITLE
Sarabun font looks different exported as PDF from Google Slide

### ISSUE THREAD

**Issue by pietrogregorini** [google/fonts#2087]
When exporting a Google Slide document in PDF using Sarabun font, it looks totally different from the original one that you can actually see while editing the Slide document.

<img width="498" alt="Schermata 2019-07-26 alle 15 13 21" src="https://user-images.githubusercontent.com/3169323/61954048-3801dc00-afb8-11e9-9775-07e023dda00b.png">
This is the expected result, which is the one shown on Google Slide

<img width="434" alt="Schermata 2019-07-26 alle 15 14 26" src="https://user-images.githubusercontent.com/3169323/61954074-44863480-afb8-11e9-8463-689f52ccd4a9.png">
This is how the font in the exported PDF looks like

**Comment by gddabe** [comment]
I hit similar issue while exporting the slides to pdf with CJK characters too
Google Slide
![image](https://user-images.githubusercontent.com/3678065/65215896-c63e9e00-dae1-11e9-8a6b-434e8042df9b.png)
PDF
![image](https://user-images.githubusercontent.com/3678065/65216034-2a616200-dae2-11e9-98d9-9ee66d99b8f9.png)

**Comment by sneko** [comment]
Same issue here with Robotto that is really fatten.

Any idea @m4rc1e @davelab6 please?

_Note: it has been mentioned on https://support.google.com/drive/thread/8422143?hl=en but no one answered this :(_

**Comment by sneko** [comment]
Any idea @m4rc1e @davelab6 please? Would be great to have your thoughts on it...

Any workaround? I'm not the only one having this, all my team experiences this sometimes...

**Comment by pietrogregorini** [comment]
@sneko I found a workaround which temporarily works, though being quite annoying and adding extra work.

I noticed that downloading the PPT of the Google Slides document, opening it with Keynote or PowerPoint (of course you have to install the used font before in your computer) you can see (more or less) the document in the right way. At least with the proper fonts. Then exporting it in PDF you will have a version with the correct fonts. This is the only way I found to work to solve this issue.

**Comment by sneko** [comment]
Yeah we already used it in the past but when having "complex" slides it breaks positioning or some other things so it's quite annoying 👎 

Defintely the idea would be to get in touch with Googlers... this issue stands for too long, and no one knows if they are aware of it... 🤔 

cc @nathan-williams @andyjgf

**Comment by pietrogregorini** [comment]
I definitely agree with you. I opened this issue more than one year ago 😉

**Comment by thlinard** [comment]
> When exporting a Google Slide document in PDF using Sarabun font, it looks totally different from the original one that you can actually see while editing the Slide document.

Looks like that Google Slides is using an old version of Sarabun when exporting PDF. Google Fonts [had this version](https://github.com/google/fonts/tree/49fbebd3dc75d42fe72c4a3eef6524f8fcc335fd/ofl/sarabun) before changing it in 2018 (PR #1651).

This is therefore an issue with Google Slides, which doesn't use the same version for display and for PDF export.

**Comment by sneko** [comment]
Hi @thlinard , do you know how we can reach the appropriate team so they investigate or provide more information?

What's weird is, 99% of time everything's working fine... but in sometimes (over the last 2 years), generations makes `Nunito Sans "Extra-Bold" and "Black"` weights more fatten than what's needed (starting from there, you just need to pray to get things back to the normal 😢 ).

So I have a doubt Google Slides uses an older version, or, it means there is some remaining fonts cached on they generating server... and since I'm not lucky I'm load-balanced to the wrong servers for the last 3 days?

Waiting & crossing fingers as it has always been... 😄

**Comment by thlinard** [comment]
> Hi @thlinard , do you know how we can reach the appropriate team so they investigate or provide more information?

Sorry, no.

> What's weird is, 99% of time everything's working fine... but sometimes (over the last 2 years), generations make `Nunito Sans "Extra-Bold" and "Black"` weights more fatten than what's needed (starting from there, you just need to pray to get things back to the normal 😢 ).
> 
> So I have a doubt Google Slides uses an older version

For Sarabun it is obvious that the PDF export uses the version before 2018 (the `g` is extremely different, for example, and corresponds well to that of the old version).

For Nunito, it could be the same problem, its passage in VF (PR #2519) [forced a rework of extreme weights](https://groups.google.com/g/googlefonts-discuss/c/sEmT8sjph24/m/C6L7BBAPEAAJ). So the observed difference could come, again, from an old version used for PDF export.

**Comment by gsgalper** [comment]
Dear Google Decision Makers,
If Google Slides will not support Nunito Sans Black or Dark Bold for PDF, why is it offered? This puts the user in a pickle, as it is not common to download for PDF. In addition, Google does not provide the share ability "view and share." One can offer view without share or editor with share. Please add a share and view combination.

**Comment by canbi** [comment]
File > Make a copy > Entire presentation. 
Then you will see printed PDF version of your presentation. Fix all padding or font issues. Then save as pdf. 

 I am not sure it is a fix for different view of the same font but it worked for padding issue.

**Comment by seoulcity** [comment]
Still not solved?

**Comment by Diego-Visualma** [comment]
Still a problem four years on.

**Comment by cvburgess** [comment]
Noto Emoji font breaks when exporting to PDF - it goes back to being tofu 😢

**Comment by adamfero** [comment]
6 years open and still not fixed @google-admin Please can we get it sorted!
